libraries/terminal/src/PrintUtilities.ts

```diff
@@ -22,40 +20,127 @@ export class PrintUtilities {
    * Returns the width of the console, measured in columns
    */
   public static getConsoleWidth(): number | undefined {
-    const stdout: tty.WriteStream = process.stdout as tty.WriteStream;
-    if (stdout && stdout.columns) {
-      return stdout.columns;
-    }
+    return process.stdout?.columns;
   }
 
   /**
-   * Applies word wrapping.  If maxLineLength is unspecified, then it defaults to the
-   * console width.
+   * Applies word wrapping.
+   *
+   * @param text - The text to wrap
+   * @param maxLineLength - The maximum length of a line, defaults to the console width
+   * @param indent - The number of spaces to indent the wrapped lines, defaults to 0
+   */
+  public static wrapWords(text: string, maxLineLength?: number, indent?: number): string;
+  /**
+   * Applies word wrapping.
+   *
+   * @param text - The text to wrap
+   * @param maxLineLength - The maximum length of a line, defaults to the console width
+   * @param linePrefix - The string to prefix each line with, defaults to ''
+   */
+  public static wrapWords(text: string, maxLineLength?: number, linePrefix?: string): string;
+  public static wrapWords(
+    text: string,
+    maxLineLength?: number,
+    indentOrLinePrefix?: number | string
+  ): string {
+    const wrappedLines: string[] = PrintUtilities.wrapWordsToLines(
+      text,
+      maxLineLength,
+      indentOrLinePrefix as string | undefined // TS is confused by the overloads
```

Comment:
This is because there isn't an overload that expressly accepts `string | number`. The last one is treated as the implementation and not one of the overloads.